package.json

```diff
@@ -45,7 +45,7 @@
     "start:all": "echo THIS IS FOR LEGACY PURPOSES ONLY. && yarn start:build",
     "start:build": "babel src -w -s -d lib --verbose --extensions \".ts,.js\"",
     "lint": "yarn lint:types && yarn lint:js && yarn lint:style",
-    "lint:js": "eslint --max-warnings 0 src test cypress",
+    "lint:js": "eslint --max-warnings 79 src test cypress # 79 enzyme deprecation warnings",
```

Comment:
that comment style isn't platform agnostic

Reply:
Replaced with `echo`